src/plone/restapi/services/navroot/configure.zcml

```diff
@@ -0,0 +1,17 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:cache="http://namespaces.zope.org/cache"
+    xmlns:plone="http://namespaces.plone.org/plone"
+    xmlns:zcml="http://namespaces.zope.org/zcml"
+    >
+
+  <plone:service
+      method="GET"
+      factory=".get.NavrootGet"
+      for="zope.interface.Interface"
+      permission="zope2.View"
+      name="@navroot"
+      />
```

Comment:
the expander needs to be also registered as an adaptor.

Reply:
For some reason I got the code there but never loaded it :)